diff --git a/compiled/watchpack/index.js b/compiled/watchpack/index.js
index 408638c9e4ae72094c06552e244353afe6d209a9..1b8c6a92f097b6938bbd3d24e7e881e64c099e90 100644
--- a/compiled/watchpack/index.js
+++ b/compiled/watchpack/index.js
@@ -9,7 +9,7 @@ module.exports = function (glob, opts) {
     throw new TypeError('Expected a string');
   }
 
-  var str = String(glob);
+  var str = String(glob).replace(/\\/g, '/');
 
   // The regexp we are building, as a string.
   var reStr = "";
